src/it/projects/json-create-lock-file-with-integrity-checking/postbuild.groovy

```diff
@@ -0,0 +1,31 @@
+import com.google.common.io.BaseEncoding
```

Comment:
I follow https://google.github.io/styleguide/javaguide.html which sorts imports alphabetically.
You can use https://github.com/vandmo/google-java-format or similar to format the files but I can do it after merging as well.

Reply:
Ran the formatter over everything and commited the results.